administrator/components/com_scheduler/tmpl/select/default.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * @package       Joomla.Administrator
+ * @subpackage    com_scheduler
+ *
+ * @copyright (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license       GNU General Public License version 2 or later; see LICENSE.txt
+ * phpcs:ignoreFile
+ */
+
+/** The SelectView default layout template. */
+
+// Restrict direct access
+defined('_JEXEC') or die;
+
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Router\Route;
+use Joomla\Component\Scheduler\Administrator\View\Select\HtmlView;
+
+/** @var  HtmlView  $this */
+
+$app = $this->app;
+
+// ? : What does this do?
+// ! : This is going down into loading the select-modal script, what does that do?
+// $function  = $app->getInput()->get('function');
+
+$wa = $this->document->getWebAssetManager();
+$wa->useStyle('com_scheduler.admin-view-select-task-css');
+$wa->useScript('com_scheduler.admin-view-select-task-search');
+
+/*
+ * if ($function) :
+ * $wa->useScript('com_scheduler.admin-select-modal');
+ * endif;*/
+?>
+
+<!-- Tasks search box on below the toolbar begins -->
+<div class="d-none" id="comSchedulerSelectSearchContainer">
+	<div class="d-flex mt-2">
+		<div class="ms-auto me-auto">
+			<label class="visually-hidden" for="comSchedulerSelectSearch">
+				<?php echo Text::_('COM_SCHEDULER_TYPE_CHOOSE'); ?>
+			</label>
+			<div class="input-group mb-3 me-sm-2">
+				<input type="text" value=""
+					   class="form-control" id="comSchedulerSelectSearch"
+					   placeholder="<?php echo Text::_('JSEARCH_FILTER'); ?>"
+				>
+				<div class="input-group-text">
+					<span class="icon-search" aria-hidden="true"></span>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+<!-- Search box and related elements end -->
+
+<div id="new-tasks-list">
```

Comment:
Just a marker, we should have another ID.

Reply:
Not sure I understand.